### PR TITLE
Fan control

### PR DESCRIPTION
**Pull request by jlashner**
Adds more in-depth fan-control to jackhammer, to allow us to set the fan-policy and fan speeds, and added docs. 
This will allow us to configure smurf to run fixed at the max fan-speed if we want, based on discussion https://github.com/simonsobs/sodetlib/discussions/354

### CODE REVIEW

File: hammers/jackhammer [hunk 1]
```diff
@@ -535,6 +518,46 @@ def activate_func(args):
 def write_env_func(args):
     write_docker_env()
 
+def setup_fans():
+    if 'smurf_fans' not in sys_config:
+        # Old way of setting up fans
+        cprint(
+            "'fans' key not found in sys config. Setting fan levels using "
```
Comment (msilvafe) [inline hammers/jackhammer:525]: Should print ‘smurf_fans’ not ‘fans’ key

File: hammers/jackhammer [hunk 2]
```diff
@@ -623,9 +646,17 @@ if __name__ == '__main__':
     write_env_parser = subparsers.add_parser('write-env', help='writes docker-env to .env file')
     write_env_parser.set_defaults(func=write_env_func)
 
+    ########### Jackhammer start-sync parser ###########
     start_sync_parser = subparsers.add_parser('start-sync')
     start_sync_parser.set_defaults(func=start_sync_func)
 
+    ########### Jackhammer start-sync parser ###########
```
Comment (msilvafe) [inline hammers/jackhammer:653]: Should this comment be setup-fans parser?

File: sodetlib/operations/uxm_relock.py [hunk 3]
```diff
@@ -260,7 +260,7 @@ def plot_channel_resonance(S, cfg, band, chan):
 
 @sdl.set_action()
 def uxm_relock(
-    S: SmurfControl, cfg, bands=None, show_plots=False,
+    S, cfg, bands=None, show_plots=False,
```
Comment (msilvafe) [inline sodetlib/operations/uxm_relock.py:263]: This seems unrelated to this PR but ok…

**Review by msilvafe — APPROVED** [review]
Minor comments. Very glad that you got this built into jackhammer. Assuming you have tested this I’m happy to merge (but would appreciate if you respond here to confirm it was tested first).

File: sodetlib/operations/uxm_relock.py (hunk 3 above)
Comment (jlashner) [reply sodetlib/operations/uxm_relock.py:263]: This was required for me to get docs to build... I may invest some time cleaning up the docs and making sure everything is up to date.

**Comment by jlashner** [comment]
Tested at site so I'm merging.

**Pull request merged**